author.tf

```diff
@@ -513,39 +513,14 @@ module "author-dynamodb" {
   slack_alert_sns_arn = "${module.eq-alerting.slack_alert_sns_arn}"
 }
 
-resource "aws_elasticache_cluster" "author-redis" {
-  cluster_id                   = "${var.env}-author-redis"
-  engine                       = "redis"
-  node_type                    = "cache.t2.micro"
-  num_cache_nodes              = 1
-  parameter_group_name         = "default.redis5.0"
-  engine_version               = "5.0.4"
-  subnet_group_name            = "${aws_elasticache_subnet_group.author-redis-subnet-group.name}"
-  security_group_ids           = ["${aws_security_group.author-redis-access.id}"]
-  availability_zone            = "eu-west-1a"
-  port                         = 6379
-}
-
-resource "aws_elasticache_subnet_group" "author-redis-subnet-group" {
-  name       = "${var.env}-author-redis-subnet-group"
-  subnet_ids = ["${module.author-vpc.database_subnet_ids}"]
-}
-
-resource "aws_security_group" "author-redis-access" {
-  name        = "${var.env}-author-redis-access"
-  description = "Redis access from the application subnet"
-  vpc_id      = "${module.author-vpc.vpc_id}"
-
-  ingress {
-    from_port   = 6379
-    to_port     = 6379
-    protocol    = "tcp"
-    cidr_blocks = "${var.author_application_cidrs}"
-  }
-
-  tags {
-    Name = "${var.env}-author-redis-security-group"
-  }
+module "author-redis" {
+  source              = "github.com/ONSdigital/eq-author-terraform-redis?ref=v1.0"
```

Comment:
I wonder if we need to introduce a new repo just yet. Nowhere else will be using it, so it might be premature to so. I think you can define the source of the module to be a local directory so could just have an author-redis directory within this repo and use that from concourse? Thoughts?

Reply:
I don't think it makes much difference. Arguably we could be doing that for all of the author specific modules i.e. dynamodb as we are never releasing these independently.

Reply:
Do you know how we reference these in concourse?